doc/radosgw/multisite.rst

```diff
@@ -852,13 +852,13 @@ The CLI will echo a JSON object with the realm properties.
         "epoch": 1
     }
 
-Use ``>`` and an output file name to output the JSON object to a file.
+Use ``>`` and a file name to redirect the output of the JSON object to a file.
 
 Set a Realm
 ~~~~~~~~~~~
 
-To set a realm, execute ``realm set``, specify the realm name, and
-``--infile=`` with an input file name.
+To set a realm, run ``realm set``, specify the realm name, and use the 
+``--infile=`` option (and provide ``--infile`` with an input file name):
```

Comment:
Seems awkward, suggest

and provide the \`\`--infile=\`\` option  with an input file name):

Reply:
I like that better.

Reply:
Actually... I don't like it better. I try to avoid the phrasal verb "to provide with" in contexts like this, because it can mean "also add x to the command" and it can also mean "when adding x to the command, remember that x requires y as an argument".

That's beside the point that you were making, which is that the definite article would make this clause a bit clearer. I agree. 

This sentence will probably be different by the time you read this.

Reply:
Perhaps something like "and add the --infile=your_input_file option" ?

Partly I think it's clear to the reader of this text that an argument named "--infile" takes an argument.